sw/host/opentitanlib/src/io/i2c.rs

```diff
@@ -60,6 +76,11 @@ pub trait Bus {
     /// 1_000_000.
     fn set_max_speed(&self, max_speed: u32) -> Result<()>;
 
-    /// Runs a I2C transaction composed from the slice of [`Transfer`] objects.
-    fn run_transaction(&self, addr: u8, transaction: &mut [Transfer]) -> Result<()>;
+    /// Sets the "default" device address, used in cases when not overriden by parameter to
+    /// `run_transaction()`.
+    fn set_default_address(&self, _: u8) -> Result<()>;
```

Comment:
Since this is the trait definition, I think we should name the parameter (e.g. `addr`).

Reply:
Done